models/intermediate/int_xero__journal_line_pivoted_tracking_categories.sql

```diff
@@ -0,0 +1,61 @@
+{{ config(enabled=(var('xero__using_journal_line_has_tracking_category', True)
+        and var('xero__using_tracking_categories', True))) }}
+
+with journal_line_has_tracking as (
+
+    select *
+    from {{ var('journal_line_has_tracking_category') }}
+),
+
+tracking_category as (
+
+    select *
+    from {{ var('tracking_category') }}
+    where lower(status) in ('active', 'archived')
+),
+
+tracking_category_option as (
```

Comment:
I noticed we don't use the `tracking_category_has_option` staging model here. I was under the impression the combination of all three models are needed to get the full tracking category picture before then joining to the journal/invoice tracking category tables.

Reply:
`journal_line_has_tracking_category` had the benefit of having both `tracking_category_id` and `tracking_category_option_id` available (unlike `invoice_line_has_tracking_category`), so I attempted a direct join using those two fields. But as discussed elsewhere it's best to standardize this with the ephemeral model gathering active tracking category/category option data.